### PR TITLE
theme + prettier [Fixes #10]

### LINKED ISSUE #10: Add different themes

### PR DESCRIPTION
**Pull request by ogroppo**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,7 +11,7 @@
     <meta property="og:title" content="$OG_TITLE" />
     <meta property="og:description" content="$OG_DESCRIPTION" />
     <meta property="og:image" content="$OG_IMAGE" />
-    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:card" content="summary_large_image" />
```
Comment (ogroppo) [inline public/index.html:14]: look what pretty things prettier does

Comment (mshd) [reply public/index.html:14]: Beautiful

**Comment by mshd** [comment]
Good work, but can you make multiple commits next time for different things

**Comment by ogroppo** [comment]
ok I will

**Pull request merged**